native/src/components/PoiDetails.tsx

```diff
@@ -6,74 +6,66 @@ import styled from 'styled-components/native'
 
 import { PoiFeature, PoiModel } from 'api-client'
 
+import ExternalLinkIcon from '../assets/ExternalLink.svg'
 import Placeholder from '../assets/PoiPlaceholderLarge.jpg'
 import useSnackbar from '../hooks/useSnackbar'
 import { getNavigationDeepLinks } from '../utils/getNavigationDeepLinks'
 import openExternalUrl from '../utils/openExternalUrl'
 import CollapsibleItem from './CollapsibleItem'
+import HorizontalLine from './HorizontalLine'
 import NativeHtml from './NativeHtml'
 import PoiDetailItem from './PoiDetailItem'
 import SimpleImage from './SimpleImage'
 
 type PoiDetailsProps = {
   poi: PoiModel
   feature: PoiFeature
-  /** define whether content will be displayed on separate detail page */
-  detailPage: boolean
-  navigateToPois: () => void
-  /** language to offer rtl support */
   language: string
 }
 
 const Thumbnail = styled(SimpleImage)`
   flex: 1;
   height: 180px;
   width: 100%;
+  border-radius: 7px;
 `
 
-const Title = styled.Text`
-  color: ${props => props.theme.colors.textColor};
-  font-family: ${props => props.theme.fonts.native.decorativeFontBold};
-  font-size: 20px;
-  padding: 32px 48px;
+const PoiDetailsContainer = styled.View`
   flex: 1;
+  padding: 0 18px;
 `
 
-const TitleContainer = styled.View`
-  flex-direction: row;
-  border-style: solid;
-  border-color: ${props => props.theme.colors.textDisabledColor};
-  border-bottom-width: 1px;
+const StyledText = styled.Text`
+  font-size: 15px;
+  font-weight: bold;
+  margin-bottom: 8px;
 `
 
-const InformationContainer = styled.View<{ detailPage: boolean }>`
-  align-items: flex-start;
-  justify-content: center;
-  border-style: solid;
-  border-color: ${props => props.theme.colors.textDisabledColor};
-  border-bottom-width: 1px;
-  border-top-width: ${props => (props.detailPage ? 0 : '1px')};
-  margin-top: ${props => (props.detailPage ? 0 : '32px')};
+const StyledDistance = styled.Text`
+  font-size: 12px;
+  margin-bottom: 16px;
 `
 
-const PoiDetailsContainer = styled.View`
-  flex: 1;
+const ExternalLink = styled.Image`
+  border-top-left-radius: 5px;
+  border-bottom-left-radius: 5px;
 `
 
-const PoiDetails: React.FC<PoiDetailsProps> = ({
-  poi,
-  feature,
-  detailPage,
-  navigateToPois,
-  language
-}: PoiDetailsProps): ReactElement => {
+const ContentWrapper = styled.View`
+  padding-right: 32px;
```

Comment:
Why so much magic number css in this file?

Reply:
As far as I know we don't have spacing classes. What should I do?

Reply:
at least using more dynamic css would be good (depending on device size)

Reply:
I think we don't do this in integreat anywhere yet, do we? So no need to start it here.

Reply:
Yes indeed we should create some spacing classes with wp f.e. like in lunes. but since we don't have that yet, i would wait for design team and refactor all paddings when we have them